cypress/support/fragments/data_import/mapping_profiles/mappingProfileDetails.js

```diff
@@ -57,6 +60,14 @@ export default {
     ]);
   },
 
+  deleteMappingProfile:(name) => {
+    cy.do([
+      Pane({ id:'full-screen-view' }).find(actionsButton).click(),
```

Comment:
Pane({ id:'full-screen-view' }) move to const var

Reply:
Done